### ISSUE TITLE
Logits and labes have different shapes when computing cross-entropy loss

### ISSUE THREAD

**Issue by pachiko** [ematvey/tensorflow-seq2seq-tutorials#33]
Hi,

I tried to implement snippets of your code for a simple word-reversal problem, where I have 3 words in a sentence but when I compute the cross-entropy it gives me an error like this: InvalidArgumentError (see above for traceback): logits and labels must be broadcastable: logits_size=[832,28] labels_size=[960,28].

I believe it is because the labels in my code have been padded whereas the logits, which are outputs of the dynamic_decode function, have variable sequence lengths. How did you manage to get the cross-entropy to work with variable sequence length for the logits?

[https://github.com/kcang2/Udacity-Deep-Learning-Assignment/blob/master/Assignment_6_3.ipynb](url)

Best regards.

**Comment by pachiko** [comment]
Sorry, mixed up your code with someone else's. Closing the issue as I don't have a problem using the dynamic_rnn object as the decoder.
